ufactory_lite6/README.md

```diff
@@ -23,6 +23,21 @@ description](https://github.com/xArm-Developer/xarm_ros2/tree/master/xarm_descri
 4. Added actuators.
 5. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
 
+### Gripper
+
```

Comment:
> The fingers can be affixed rotated internally

Typo?

Reply:
In the sense of "affixed with an internal rotation" but I'll rewrite it all to be clearer

Reply:
Gotcha!